### DISCUSSION TITLE
installazione di test

### DISCUSSION THREAD

**Discussion by Nik71git** [maxill1/ialarm-mqtt#48] [General]
Ciao, uso con soddisfazione l'addon si HA tramite broker Mosquitto (broker A), ho anche una seconda installazione di HA istanziata su docker con un broker (brocker B, sempre Mosquitto) sempre su docker. 
Ora vorrei provare l'installazione di iAlarm-mqtt su docker usando il broker B.
La domanda è: essendo la centrale unica ci possono essere problemi se viene consultata da due broker distinti contemporaneamente?
La domanda nasce perché l'obiettivo è avere una istanza di HA di backup ma dockerizzata, solo core e quindi senza addon.

**Comment by mrMiimo** [comment]
Queste centraline non gestiscono molto bene il flusso di rete… però se hai problemi al limite spegni la in stanza

**Comment by maxill1** [comment]
> Ciao, uso con soddisfazione l'addon si HA tramite broker Mosquitto (broker A), ho anche una seconda installazione di HA istanziata su docker con un broker (brocker B, sempre Mosquitto) sempre su docker. Ora vorrei provare l'installazione di iAlarm-mqtt su docker usando il broker B. La domanda è: essendo la centrale unica ci possono essere problemi se viene consultata da due broker distinti contemporaneamente? La domanda nasce perché l'obiettivo è avere una istanza di HA di backup ma dockerizzata, solo core e quindi senza addon.

Se la tua installazione principale usa un broker (broker A) e la secondaria un altro (broker B) non dovrebbero esserci problemi di entities duplicate o con stati incoerenti.

L'incognita grande è "come reagirà la centrale alla doppia connessione TCP _persistente_?" (addon HA e istanza ialarm-mqtt su docker). In un certo senso è come se tu avessi l'addon di HA connesso fisso alla centrale e allo stesso tempo aperta l'applicazione android, se non ti da problemi, dovrebbe funzionare.

Queste centrali sebbene simili tra loro reagiscono in modo diverso (una si impalla, l'altra no, etc), l'unico modo è provare.
